worlds/timespinner/Options.py

```diff
@@ -418,6 +469,11 @@ class TimespinnerOptions(PerGameCommonOptions, DeathLinkMixin):
     trap_chance: TrapChance
     traps: Traps
 
+class HiddenBossRandoOverrides(BossRandoOverrides):
```

Comment:
Since this a new setting you dont need to add it to the `BackwardsCompatiableTimespinnerOptions`

Reply:
Ah cool I was wondering about that